documentation/example.yml

```diff
@@ -41,7 +41,7 @@ Network:
     # the self-signed cert will be created at $GENISYS_CERT_STORE or /etc/genisys/ssl/
     # if the ssl section is missing, the server will be ran without ssl
     ssl:
-      cert: /path/to/fullchain.pem
+      cert: "../genisys/modules/ssl/cert.pem"
```

Comment:
I'm assuming this was changed for testing. Please keep test configs in the tests/ directory so that we can keep this file as a piece of documentation.